backend/python/app/services/implementations/story_service.py

```diff
@@ -267,6 +268,30 @@ def update_story_translation_contents(self, story_translation_contents):
             )
             raise error
 
+    def update_story_translation_stage(self, story_translation_data):
+        story_translation = StoryTranslation.query.filter_by(
```

Comment:
should we wrap this in a try except (like the other functions above) so we catch and log exceptions?

Reply:
do we need to cast the error to string before logging? `self.logger.error(str(error))` (we seem to do this for most of the other functions)